tests/unit/test_vmware_exporter.py

```diff
@@ -116,6 +119,34 @@ def test_collect_vms():
     }
     assert metrics['vmware_vm_guest_disk_capacity'].samples[0][2] == 100
 
+    # VM tools info (vmguest)
+    assert metrics['vmware_vm_guest_tools_running_status'].samples[0][1] == {
+        'vm_name': 'vm-1',
+        'host_name': 'host-1',
+        'cluster_name': 'cluster-1',
+        'dc_name': 'dc',
+        'tools_status': 'toolsOk',
+    }
+    assert metrics['vmware_vm_guest_tools_running_status'].samples[0][2] == 1.0
+
+    assert metrics['vmware_vm_guest_tools_version'].samples[0][1] == {
+        'vm_name': 'vm-1',
+        'host_name': 'host-1',
+        'cluster_name': 'cluster-1',
+        'dc_name': 'dc',
+        'tools_version': '10336',
+    }
+    assert metrics['vmware_vm_guest_tools_version'].samples[0][2] == 1.0
+
+    assert metrics['vmware_vm_guest_tools_version_status'].samples[0][1] == {
+        'vm_name': 'vm-1',
+        'host_name': 'host-1',
+        'cluster_name': 'cluster-1',
+        'dc_name': 'dc',
+        'tools_version_status': 'guestToolsUnmanaged',
```

Comment:
tools_status or tools_version_status?

Reply:
If the api reveals it as tools_version_status. I say leave it.

Reply:
There are 3 different attributes:
1) `tools_running_status` - indicates if the tools is running or not (i.e. `toolsOk`)
2) `tools_version` - the version of the installed tools
3) `tools_version_status` - whether it is managed on unmanaged deployed (i.e. if it's official vmware tools or the open-vm-tools)

I'm open to changing the label/metric names if you think it's confusing, but seems to be quite aligned with the api.

We could potentially even merge this into a single metric, but to be able to safe guard each value, I suppose it's better to keep them separate.

Reply:
I'd rather have it match the api tbh